addons/advanced_throwing/XEH_postInit.sqf

```diff
@@ -63,6 +66,13 @@ GVAR(ammoMagLookup) = call CBA_fnc_createNamespace;
     [_this select 1, "Player changed"] call FUNC(exitThrowMode);
 }] call CBA_fnc_addPlayerEventhandler;
 
+["visiblemap", {
+    if (visibleMap && {ACE_player getVariable [QGVAR(inHand), false]}) then {
```

Comment:
Adding the params call is slower that just using command and global var
0.00286 ms vs 0.00412 ms